src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj

```diff
@@ -15,10 +15,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" Condition="'$(IncludeApplicationInsights)' == 'True'" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" Condition="'$(Framework)' != 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.4" Condition="'$(Framework)' == 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.4" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.5" Condition="'$(Framework)' == 'netcoreapp1.1'" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.6" />
```

Comment:
This file looks odd compared to the other files. Every other project has two separate dependency sections for 1.0 and 1.1. However, the only PackageReference that's different between 1.0 and 1.1 for this project is Microsoft.AspnetCore. Does this mean that the 1.0 fsproj also references Microsoft.AspNetCore.Mvc 1.1.6 and Microsoft.Extensions.Logging.Debug 1.1.2?

Reply:
I verified that this looks really wrong by using the latest 1.1.5 sdk from dot.net. I ran the command `dotnet.exe new webapi -lang F# --framework netcoreapp1.0` and it gave me this fsproj:
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Controllers/*.fs" />
    <Compile Include="Startup.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.4" />
    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />

  </ItemGroup>
  
</Project>
```

This is not a supported configuration (mix of 1.0 and 1.1 packages).

Reply:
@DamianEdwards FYI